src/main/build/debug.h

```diff
@@ -55,7 +55,9 @@ typedef enum {
     DEBUG_GYRO_NOTCH,
     DEBUG_RC_INTERPOLATION,
     DEBUG_ANGLERATE,
+#ifdef USE_DSHOT
```

Comment:
I thought the debug modes were indicated by index in the blackbox logs, and then resolved in the log viewer? If so, this will make it very hard to resolve log dimensions in the blackbox log viewer. @basdelfos would probably know.

Reply:
You are right. In the log file there is only the index, not the name.